### DISCUSSION TITLE
Defining "Path:'*/sites/HR/*'" inside my PnP -SearchResult Query temaplte >> will show documents from all sites and not only from the HR site.. any advice?

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2815] [Q&A]
I define the following Query Template to show Documents only from /sites/HR >> but currently i am getting documents from all the sites:-

![image](https://user-images.githubusercontent.com/79726511/222147108-8d9d1008-986b-42f9-a261-a0179774342e.png)

Also if i define random characters >> i will still get all documents.. so seems the Query Template is been ignored in a way or another.

![image](https://user-images.githubusercontent.com/79726511/222147341-62760406-58b7-4736-a103-012e5c23b35c.png)

any advice? please?

**Comment by Parmstig** [comment]
Could you try with actual url without the wildcard in the beginning.

**Reply by mvcsharepointdev** [reply]
@Parmstig yes full url worked,,so we can not use wildcard inside Query Template?

**Reply by kasperbolarsen** [reply]
Sure you can, but KQL only supports "begins with" not "ends with" please see https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference

**Reply by wobba** [reply]
And path take a full URL prefix, and is specially handled.
